source-map.bs

```diff
@@ -357,8 +357,8 @@ The generated code should include a line at the end of the source, with the foll
 ```
 
 Note: The prefix for this annotation was initially `//@` however this conflicts with Internet
-Explorer's Conditional Compilation and was changed to `//#`.  It is reasonable for tools to
-also accept `//@` but `//#` is preferred.
+Explorer's Conditional Compilation and was changed to `//#`. Source map generators should only support `//#` 
+while source map consumers should support both `//@` and `//#` for legacy compatibility. 
```

Comment:
```suggestion
Explorer's Conditional Compilation and was changed to `//#`. Source map generators must
only emit `//#`, while source map consumers must accept both `//@` and `//#`.
```

_must_, not _should_, as the spec should be as assertive as possible :)

Also, bikeshed supports notes that get formatted as such. They are usually used to describe the reason for something or to give additional information that is not a requirement for implementations. I think we could write the legacy reason in its own paragraph as ``Note: `//@` is needed for compatibility with some existing legacy source maps.``

Reply:
Thanks. I've updated the comments.